src/modules/rolle/api/dbiam-rolle.error.ts

```diff
@@ -0,0 +1,21 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { DbiamError, DbiamErrorProps } from '../../../shared/error/dbiam.error.js';
+
+export enum RolleErrorI18nTypes {
+    ROLLE_ERROR = 'ROLLE_ERROR',
+    ADD_SYSTEM_RECHT_ERROR = 'ADD_SYSTEM_RECHT_ERROR',
```

Comment:
```suggestion
    ROLLE_ERROR = 'ROLLE_SPECIFICATION_ERROR',
    ADD_SYSTEM_RECHT_ERROR = 'ADD_SYSTEMRECHT_ERROR',
```
Vorschlag für Einheitlichkeit

Reply:
werde ich anpassen